### ISSUE TITLE
Icon for "allow/disallow student upload" is confusing

### ISSUE THREAD

**Issue by Sturek** [elan-ev/studip-opencast-plugin#872]
The icon represents the current state of the setting, the text represents the change of the state when the button is pressed. This is incongruent and confusing. I would suggest either changing the icon to represent the action of the button (this would fit the other buttons as well). On the other hand, this means there is no icon representing the current state. Maybe this should be discussed. What do you think @kai-wng-uos ?

![Image](https://github.com/elan-ev/studip-opencast-plugin/assets/6900803/af3b4d30-6b15-474c-a9e4-0451b913a407)

**Comment by kai-wng-uos** [comment]
I agree, I think changing the changing the button would work. Alternatively, maybe we could change the text to represent the current state, as in "Studierendenupload erlaubt" vs. "Studierendenupload nicht erlaubt".

**Comment by rlucke** [comment]
Encrypting the status via an icon and offering the reverse action next to it is not a good ux. According to Stud.IP design guidelines, the icons in the sidebar should reflect the activity. The icon should not be an indicator for a status. A completely different solution is needed for this function, outside the sidebar.
